loader_hub/whatsapp/base.py

```diff
@@ -0,0 +1,57 @@
+"""Whatsapp chat data loader"""
+
+from pathlib import Path
+from typing import List
+
+from gpt_index.readers.base import BaseReader
+from gpt_index.readers.schema.base import Document
+
+
+class WhatsappChatLoader(BaseReader):
```

Comment:
You'll need to add this to the `library.json` file as well in order for it to be picked up by `download_loader` in LlamaIndex and the LlamaHub website.

Reply:
Missed this! Thanks